blog/2024-05-07-weaviate-1-25-release/_core-1-25-include.mdx

```diff
@@ -40,8 +44,22 @@ Above the threshold value, HNSW indexes are faster for queries, but they also ha
 
 Dynamic indexes are particularly useful in a multi-tenant setup. The resource costs are high to build an HNSW index for every tenant. However, if a tenant collection grows large enough, the index dynamically switches to HNSW. The smaller tenants continue to use flat indexes.
 
+## Raft
+
+import WV8onRaft from './img/raft-weaviate.png';
+
+<center><img src={WV8onRaft} width="50%" alt="Weaviate on a raft"/></center>
+
+Weaviate clusters can be large; there can be a lot of nodes to coordinate. The host systems have to work together reliably and efficiently, even under high loads. [Raft](https://raft.github.io/) is a robust consensus algorithm that helps make Weaviate faster and more fault tolerant. 
+
+There are two types of data in a Weaviate cluster: your actual data, and system state information. Earlier releases use the same mechanism to store both types of data. Starting in 1.25, Weaviate uses Raft to store schema information and cluster state details. The underlying data [storage](/developers/weaviate/concepts/storage) is managed the same way as before - [replication](/developers/weaviate/concepts/replication-architecture) and [sharding](/developers/weaviate/concepts/cluster) continue to safeguard your data while making it available for your applications. Raft ensures your schemas are safe and helps you to reliably scale your production workloads. 
```

Comment:
Earlier releases use the same mechanism to store both types of data. Starting in 1.25, Weaviate uses Raft to store schema information and cluster state details.

What about:
Earlier releases used two-phase commits to achieve strong consistency for the schema. Starting in 1.25, Weaviate uses Raft to improve the cluster's fault tolerance when it comes to the schema.